src/OrleansServiceBus/Providers/Streams/EventHub/EventHubStreamProviderSettings.cs

```diff
@@ -17,6 +17,37 @@ public class EventHubStreamProviderSettings
         public string StreamProviderName { get; }
 
         /// <summary>
+        /// SlowConsumingMonitorFlowControlThresholdName
+        /// </summary>
+        public const string SlowConsumingMonitorFlowControlThresholdName = "SlowConsumingMonitorFlowControlThreshold";
```

Comment:
nameof(SlowConsumingMonitorFlowControlThreshold)
Avoid hardcoding strings in several places in this class. We were probably doing this before we upgraded to c# 6, but there's no reason to do more hardcoding now